src/config/dynamic-theme-fixes.config

```diff
@@ -3967,6 +3981,33 @@ a[data-mentionid],
 
 ================================
 
+forums.laptopvideo2go.com
```

Comment:
I just noticed they share the same selectors, so might be worth removing this one and add
`forums.laptopvideo2go.com`
to Line 3985, like:
```css
forum.eset.com
forums.laptopvideo2go.com
```